login/tests.py

```diff
@@ -0,0 +1,3 @@
+# from django.test import TestCase
+
+# Create your tests here.
```

Comment:
write the relevant issue for those tests

Reply:
does it need to be in a separate issue or is it ok to edit the issue was specified in the PR content?

Reply:
I think that it needs to be a separate issue